docs/java/how-to-develop-a-workflow-definition-in-java.md

```diff
@@ -50,19 +47,106 @@ public interface FileProcessingWorkflow {
 }
 ```
 
-The `@WorkflowMethod` annotation has a `name` parameter, such as `@WorkflowMethod(name = "MyWorkflowType")`.
-It can be used to denote the [Workflow Type](/docs/concepts/what-is-a-workflow-type). If not set, the Workflow Type defaults to the short name of the Workflow interface. In the example above, the Workflow Type defaults to `FileProcessingWorkflow`.
+### `@WorkflowMethod`
+
+The `@WorkflowMethod` identifies the method that is starting point of the Workflow Function Execution. The [Workflow Execution](/docs/concepts/what-is-a-workflow-execution) completes when this method completes.
+A Workflow Definition interface in Java can only have one method annotated with `@WorkflowMethod`. It can be used to denote the [Workflow Type](/docs/concepts/what-is-a-workflow-type).
+The Workflow Type defaults to the short name of the Workflow interface. In the following example, the Workflow Type defaults to "NotifyUserAccounts".
+
+```java
+  @WorkflowInterface
+
+  public interface NotifyUserAccounts {
+    @WorkflowMethod
+    void notify(String[] accountIds);
+}
+```
+
+To overwrite this default naming and assign a custom Workflow Type, use the `@WorkflowMethod` annotation with the `name` parameter. In the following example, the Workflow Type is set to “Abc”.
+
+```java
+@WorkflowInterface
+
+  public interface NotifyUserAccounts {
+  @WorkflowMethod(name = "Abc")
+  void notify(String[] accountIds);
+  }
+```
+
+When setting the Workflow Type this way, the value of the `name` parameter does not have to start with an upper case letter.
```

Comment:
```suggestion
When you set the Workflow Type this way, the value of the `name` parameter does not have to start with an uppercase letter.
```
First edit avoids a dangling participle. (Otherwise, "the value" sets the Workflow Type rather than the developer.)